tfjs/rollup.config.js

```diff
@@ -133,6 +133,7 @@ module.exports = cmdOptions => {
     },
     tsCompilerOptions: {target: 'es5'},
     external: [
+      'long',
```

Comment:
Does this mean we're not including `long` in the bundle? Do we need it in the browser?